dm/docs/RFCS/20220124_enhance_task_manageability.md

```diff
@@ -0,0 +1,131 @@
+| get      | `dmctl source get --output="source1.yaml" source1` | --output(default: ".")  | source-name              | show the source config in yaml format, also support output to file. |
+| list     | `dmctl source list`                                |                         |                          | list all source in current cluster.                                 |
+| status   | `dmctl source status source1`                      |                         | source-name              | show source detail status.                                          |
+| enable   | `dmctl source enable source1`                      |                         | source-name              | enable a disabled source.                                           |
+| disable  | `dmctl source disable source1`                     |                         | source-name              | disable a source and also stop the running subtasks of this source. |
+| transfer | `dmctl source transfer source1 worker1`            |                         | source-name, worker-name | transfers a source to a free worker.                                |
+
+### dmctl commands for Relay
+
+| Command | Full Syntax Example                                                                                 | Flags                      | Arguments              | Description                                                                  |
+|---------|-----------------------------------------------------------------------------------------------------|----------------------------|------------------------|------------------------------------------------------------------------------|
+| start   | `dmctl relay start --worker-name="worker1" source1`                                                 | --worker-name(default: "") | source-name            | start relay for a source on a worker.                                        |
+| stop    | `dmctl relay stop --worker-name="worker1" source1`                                                  | --worker-name(default: "") | source-name            | stop relay for a source on a worker.                                         |
+| purge   | `dmctl relay purge --sub-dir="2ae76434-f79f-11e8-bde2-024ac130008.000001" source1 mysql-bin.000006` | --sub-dir(default: "")     | source-name, file-name | purges relay log files of the DM-worker according to the specified filename. |
+
+### dmctl commands for DDL-LOCK
+
+| Command | Full Syntax Example                 | Flags | Arguments          | Description                                  |
+|---------|-------------------------------------|-------|--------------------|----------------------------------------------|
+| list    | `dmctl ddl-lock list task1`         |       | task-name          | show shard-ddl locks information for a task. |
+| unlock  | `dmctl ddl-lock unlock task1 lock1` |       | task-name, lock-id | force unlock un-resolved DDL locks.          |
```

Comment:
in my memory the task-name is a part of lock-id, currently the `unlock-ddl-lock` command doesn't need task name